### PR TITLE
User_Profile_Page is Added

### PR DESCRIPTION
**Pull request by AkshayJyani**
User Profile Section is now added

### CODE REVIEW

**Comment by Anshultilondiya** [comment]
- [ ] Add a description of User Profile Page 
- [ ] Add screenshot of work
- [ ] Transfer Your code to UserProfile.js { Your Code is in User.js in Components UserProfile page is not a component therefore it should be in admin/UserProfile.js 
- [ ] Add route to your page
- [ ] Rebase before making PR

**Comment by Anshultilondiya** [comment]
@AkshayJyani Resolve Conflict of App.css File

**Comment by AkshayJyani** [comment]
I have done 
I am attaching screenshot image it's not perfect because due to some bug main site is not working.

File: frontend/src/components/profiles/UserProfile.js [hunk 1]
```diff
@@ -1,14 +1,76 @@
 import React from "react";
-import User from "./User/User.js";
+// import User from "./User/User.js";
+import "../../App.css";
```
Comment (Anshultilondiya) [inline frontend/src/components/profiles/UserProfile.js:3]: App.css styles globally no need to import in any file other than App.js

**Comment by Anshultilondiya** [comment]
Why main site is not working ?

File: frontend/src/components/profiles/UserProfile.js (hunk 1 above)
Comment (AkshayJyani) [reply frontend/src/components/profiles/UserProfile.js:3]: okay I will update it

**Comment by neeraj-2** [comment]
> Why main site is not working ?

@Anshultilondiya I have created a PR for the same.

File: frontend/src/components/profiles/UserProfile.js [hunk 2]
```diff
@@ -32,35 +30,35 @@ function UserProfile() {
                     <div className="active_user_items">
                         <p>Active</p>
                         <div className="list_active_user_items">
-                            <pre className="tab">Item_name               Issue_date:             Return_date:              Due_date: </pre>
+                            <pre className="tab">Item_name Issue_date: Return_date: Due_date: </pre>
```
Comment (Anshultilondiya) [inline frontend/src/components/profiles/UserProfile.js:33]: why <pre> 
how you will add name, date and due date
don't use pre tag 
use dummy to fill the fields 
make separate tags for name, date and due date

**Review by Anshultilondiya — CHANGES_REQUESTED** [review]
remove all the pre tag and make separate tags for all the requested fields 
Use some dummy data for filling

**Comment by AkshayJyani** [comment]
I have pull the changes as well as made suitable change and also resolve request for changes.
Screenshot (301)
Screenshot (302)

**Review by Anshultilondiya — APPROVED** [review]
Add Stateful rendering of data

**Pull request merged**